Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM rust:1.42.0
+ENV KUBECTL_VERSION=v1.17.3
+ENV CLOUD_SDK_VERSION=283.0.0
+ENV INTERNAL_IP=127.0.0.1
+ENV EXTERNAL_IP=0.0.0.0
+
+RUN set -ex \
+    && apt-get clean \
+    && apt-get update \
+    && apt-get upgrade -y \
+    && apt-get install -y --no-install-recommends git apt-utils apt-transport-https ca-certificates
+
+RUN wget https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/amd64/kubectl && \
+  chmod a+x ./kubectl && \
+  mv ./kubectl /usr/local/bin
+ENV PATH /google-cloud-sdk/bin:$PATH
+RUN cd / && curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${CLOUD_SDK_VERSION}-linux-x86_64.tar.gz && \
```

Comment:
Why do we need gcloud in the image?

Reply:
pod needs access to run kubectl to get LB ip. Further on (and I have told them ab it) if for whatever reason any of the ips update nym will stop working. I told them that they need to use port binging and external discovery.

Reply:
Sure kubectl is ok, i was asking about cloud in the container, we are not using it.